### PR TITLE
Roll-back of multiple TcpServers

### LINKED ISSUE #649: Client location sensing for proper URI preparation at server side
Is it necessary to add an additional function to the ApplicationPattern for "sensing" the location of the client for being able to provide proper URI e.g. in the request of the Generic Representation?

x-Domain communication must be supported, too. 

(Issue is put into 2.0.2_spec for keeping it in mind. Most likely it will be transferred into 2.1.0.)

### PR DESCRIPTION
**Pull request by openBackhaul**
Fixes #649

### CODE REVIEW

**Comment by openBackhaul** [comment]
Hi Prathiba, thanks for commenting. If correctly solved, please not just comment, but also merge feature branch (openBackhaul/issue649) into release branch (AP-v2.0.2_spec). Best regards, Thorsten

File: spec/ApplicationPattern.yaml [hunk 1]
```diff
@@ -726,6 +690,8 @@ paths:
                             from [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-op-s-bm-011/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-capability/operation-name]'
                         tcp-server-list:
                           type: array
+                          minItems: 1
+                          maxItems: 1
```
Comment (PrathibaJee) [inline spec/ApplicationPattern.yaml:694]: Also in the example , the tcp-server-list shall contain a single entry. (line no 736)

File: spec/ApplicationPattern+services.yaml [hunk 2]
```diff
@@ -55,11 +55,6 @@ servers:
       own-ip-address: ## TODO: Add OwnIpAddress
```
Comment (PrathibaJee) [inline spec/ApplicationPattern+services.yaml:55]: tcp-server is represented as a list(a hyphen in the starting). Similar to http-server , this shall be represented as an object(without hyphen)

**Review by PrathibaJee — CHANGES_REQUESTED** [review]

File: spec/ApplicationPattern.yaml (hunk 1 above)
Comment (openBackhaul) [reply spec/ApplicationPattern.yaml:694]: Of course. Deleted the https tcp-server.

File: spec/ApplicationPattern+services.yaml (hunk 2 above)
Comment (openBackhaul) [reply spec/ApplicationPattern+services.yaml:55]: Well actually, the modelling is still a list, despite we agreed to put just one instance of the object.
If you would agree, I would prefer leaving that one as is until we changed the modelling back towards a single object.

Comment (PrathibaJee) [reply spec/ApplicationPattern+services.yaml:55]: Sure , let's leave it as it is. Thanks.

**Review by PrathibaJee — APPROVED** [review]

**Pull request merged**